### ISSUE TITLE
Loadout name localisation is missing for all roles

### ISSUE THREAD

**Issue by UnicornOnLSD** [RMC-14/RMC-14#8019]
## Description

Title, localisation is missing, likely just forgotten to do.

**Reproduction**

Open customisation menu, check loadout window, look at window name
**Screenshots**

<img width="2560" height="1440" alt="Image" src="https://github.com/user-attachments/assets/32f4db61-8477-4a66-85c3-89bed600b829" />

**Additional context**
